web-ui/src/pages/ResourcesPage.jsx

```diff
@@ -22,52 +13,65 @@ const ResourcesPage = () => {
   const ociLightBlue = "#72c7d5";
   const ociOrange = "#feb672";
 
+  const getPDF = async (name) => {
+    try {
+      const res = await fetch("http://localhost:3000/pdfs/" + name + ".pdf");
```

Comment:
why axios sometimes and fetch others? and this http://localhost:3000 stuff need to be removed, please.